lib/macro.h

```diff
@@ -553,8 +553,9 @@ int out_len = 0;\
         _06_08_ENCODE_DROPS_FEE            (fee_ptr, fee                            );                               \
     }
 
-
-
 #endif
 
+#define GET_HOOKSET_OPERATION_SIZE(operation_type)\
+    (operation_type == 1U ? 75 : operation_type == 2 ? 9 : 2)
```

Comment:
It is better if you can introduce these figures using constants. (with proper names)